consumer/src/app/(purchase)/carrinho/page.tsx

```diff
@@ -9,6 +9,7 @@ import { useEffect, useState } from "react";
 import { useCartProvider } from "../../../context/cart";
 import { RiArrowDownSLine } from "react-icons/ri";
 import { RiArrowUpSLine } from "react-icons/ri";
+import { OfferDTO } from "@shared/interfaces/dtos";
```

Comment:
Essa importação não está sendo usada no arquivo. Pode remover se não for ser usada.